ChangeLog

```diff
@@ -1,3 +1,35 @@
+3.12.1:
+	- Add commit title to changelog by default, if it has ticket number
+	  (ENT-3638)
```

Comment:
This change isn't really user facing, and can be removed imo.

Reply:
well, it concerns those who are committing:
https://github.com/cfengine/core/blob/master/HACKING.md#changelog-entries

Reply:
> When a new feature or a bugfix is being merged, it is often necessary to be accompanied by a proper entry in the ChangeLog file.

> Keep in mind that changelog entries should be written in a way that is understandable by non- programmers. This means that references to implementation details are not appropriate, leave this for the non-changelog part of the commit message. It is the behavior change which is important. This implies that refactorings that have no visible effect on behavior don't need a changelog entry.

IMO, and based on the quotes above, you should edit HACKING.md with this change, and not have it in the Changelog. A change in how changelogs are generated is A) not a bugfix or feature, B) not relevant to users/non-programmers.

Reply:
agreed